packages/patternfly-4/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/FormControl/form-control.css';
+import { css } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils';
+import { FormEvent } from 'react';
+
+export enum inputTypes {
```

Comment:
Would be better to prefix with the component name so as not to cause collisions in the future, e.g. `TextInputTypes`